app/src/main/java/com/example/cupcake/CupcakeScreen.kt

```diff
@@ -75,8 +95,40 @@ fun CupcakeApp(
                 navigateUp = { /* TODO: implement back navigation */ }
             )
         }
-    ) { innerPadding ->
+        ) { innerPadding ->
         val uiState by viewModel.uiState.collectAsState()
-
+        NavHost(
+            navController = navController,
+            startDestination = CupcakeScreen.Start.name,
+            modifier = Modifier.padding(innerPadding)
+        ) {
+            composable(route = CupcakeScreen.Start.name){
+                StartOrderScreen(
+                    quantityOptions = DataSource.quantityOptions,
+                    onNextButtonClicked = {},
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .padding(dimensionResource(R.dimen.padding_medium))
+                )
+            }
+            composable(route = CupcakeScreen.Flavor.name) {
+                val context = LocalContext.current
+                SelectOptionScreen(
+                    subtotal = uiState.price,
+                    options = DataSource.flavors.map { id -> context.resources.getString(id) },
+                    onSelectionChanged = { viewModel.setFlavor(it) },
```

Comment:
```suggestion
                    onSelectionChanged = viewModel::setFlavor,
```
This is a nice syntactic sugar